lib/rex/proto/kerberos/model/element.rb

```diff
@@ -30,6 +30,8 @@ def self.decode(input)
           end
 
           def initialize(options = {})
+            raise ArgumentError, "Invalid options, expected hash, got #{options.class}" unless options.is_a?(Hash)
```

Comment:
Added this to let folk know you can't do `KerberosObject.new(asn1)`. Happy to revert.